src/main/java/talkabot/task/Task.java

```diff
@@ -0,0 +1,63 @@
+package talkabot.task;
+
+/**
+ * Task class contains task description and completion status.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Constructor for Task class.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Returns status icon of task.
+     *
+     * @return Status icon.
+     */

```

Comment:
I thought this part can be slightly more specific? Maybe something like giving an example, like e.g. if the task is done, return the status icon X to represent the marking of task. I thought it seems more clear this way when users see the Javadoc. 👍